zapisy/apps/grade/poll/templates/poll/results.html

```diff
@@ -70,50 +70,8 @@ <h5 class="card-title">{{ entry.question }}</h5>
         {% if semester.id == selected_semester.id  %}selected{% endif %}>{{ semester }}</option>
       {% endfor %}
     </select>
-    <div class="accordion" id="course-sections">
-      {% for group_name, entries in polls.items %}
-      {% with index=iterator|next %}
-      <div class="card">
-        <div class="card-header" style="cursor: pointer;" id="course-section-{{ index }}-heading" data-toggle="collapse"
-          data-target="#course-section-{{ index }}" aria-expanded="false" aria-controls="course-section-{{ index }}">
-          <div class="d-flex w-100 justify-content-between">
-            <span>{{ group_name }}</span>
-            <span class="text-right text-nowrap">
-              {{ submissions_count|lookup:group_name|default_if_none:0 }}
-            </span>
-          </div>
-        </div>
-        <div
-          class="border-top collapse {% if current_poll and current_poll.category == group_name %}show{% endif %} list-group list-group-flush"
-          id="course-section-{{ index }}" aria-labelledby="course-section-{{ index }}-heading">
-          {% for entry in entries %}
-          {% if current_poll and entry.id == current_poll.id %}
-          <a href="{% url 'grade-poll-results' semester_id=selected_semester.id poll_id=entry.id %}"
-            class="list-group-item list-group-item-action active">
-            <div class="d-flex w-100 justify-content-between">
-              <span>{{ entry.subcategory }}</span>
-              <span class="text-right text-nowrap">{{ entry.number_of_submissions }}</span>
-            </div>
-          </a>
-          {% else %}
-          <a href="{% url 'grade-poll-results' semester_id=selected_semester.id poll_id=entry.id %}"
-            class="list-group-item list-group-item-action">
-            <div class="d-flex w-100 justify-content-between">
-              <span>{{ entry.subcategory }}</span>
-              <span class="text-right text-nowrap">{{ entry.number_of_submissions }}</span>
-            </div>
-          </a>
-          {% endif %}
-          {% endfor %}
-        </div>
-      </div>
-      {% endwith %}
-      {% empty %}
-      <div class="alert alert-info">
-        Brak przesłanych ankiet w wybranym semestrze.
-      </div>
-      {% endfor %}
-    </div>
+    <div class="accordion" id="course-sections" data-polls="{{ polls | to_json }}" data-submissions-count="{{ submissions_count | to_json }}" data-current-poll="{{ current_poll | to_json }}" data-selected-semester-id="{{ selected_semester.id }}" data-is-superuser="{{ is_superuser | to_json }}"></div>
```

Comment:
A tu może żaden linter nie marudzi, ale naszym białkowym aparatem poznawczym sami możemy ocenić, że ten wiersz warto trochę połamać.

Reply:
No i ta dana nie jest żadnym złożonym obiektem chyba:
```suggestion
    <div class="accordion" id="course-sections" data-polls="{{ polls | to_json }}" data-submissions-count="{{ submissions_count | to_json }}" data-current-poll="{{ current_poll | to_json }}" data-selected-semester-id="{{ selected_semester.id }}" data-is-superuser="{{ is_superuser }}"></div>
```

Reply:
Złożonym obiektem nie jest, ale pythonowe True != js'owe true.
`Error in render: "SyntaxError: Unexpected token 'T', "True" is not valid JSON"`